homeassistant/components/sfr_box/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up SFR box as config entry."""
-    box = SFRBox(ip=entry.data[CONF_HOST], client=get_async_client(hass))
+    box = SFRBox(ip=entry.data[CONF_HOST], client=async_get_clientsession(hass))
```

Comment:
Note: this change is compulsory due to the amends in the underlying library (httpx > aiohttp)